doc/en/user/source/security/passwd.rst

```diff
@@ -97,8 +97,14 @@ By default, the master password is generated and stored in a file named ``securi
 
 .. warning:: The file ``security/masterpw.info`` is a security risk. The administrator should read this file and verify the master password by logging on GeoServer as the ``root`` user. On success, this file should be removed.
 
+.. warning:: First thing to do as an Administrator of the System, would be to dump the Master Password generated by GeoServer, store it somewhere not accessible by anyone, and delete any ``security/masterpw.info`` or whatever file you used to dump the password in clear.
+
 Refer to :ref:`security_webadmin_masterpasswordprovider` for information on how to change the master password.
 
+.. note:: By default the login to Admin GUI and REST APIs with Master Password is disabled. In order to enable it you will need to manually change the Master Password Provider ``config.xml``, usually located into ``security/masterpw/default/config.xml``, by adding the following statement::
```

Comment:
I would suggest the following wording changes:

> By default login to the Admin GUI and REST APIs using the Master Password is disabled. In order to enable it you will need to manually change the Master Password Provider ``config.xml``, usually located in ``security/masterpw/default/config.xml``, by adding the following statement::

Reply:
I should have updated it accordingly to your notes. Thanks for reviewing it.